inc/saivlan.h

```diff
@@ -452,6 +452,58 @@ typedef enum _sai_vlan_member_attr_t
      */
     SAI_VLAN_MEMBER_ATTR_VLAN_TAGGING_MODE,
 
+    /**
+     * @brief Indicates if the bridge port is set to drop the broadcast, unknown unicast and multicast traffic
+     *
+     * When set to true, ingress BUM traffic will be dropped
+     * Valid only when the SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID is of type PORT.
```

Comment:
For these new attributes, can we indicate the conditions using "@Validonly" flag ?

Reply:
SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID  is an oid. So I am not clear how one can formalise this in a @validonly statement.

Reply:
> SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID is an oid. So I am not clear how one can formalise this in a @validonly statement.

Looks like SAI infra has no way to mention this. We can go by comment description. Thanks.